packages/charts/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx

```diff
@@ -443,6 +443,7 @@ export class VerticalStackedBarChartBase
             opacity={shouldHighlight ? 1 : 0.1}
             strokeWidth={3}
             strokeLinecap="round"
+            strokeDasharray={this._points[i].lineData?.[0]?.lineOptions?.strokeDasharray}
```

Comment:
lineOptions may vary depending on lineData. Ensure that you use the correct indices or consider using lineObject.